boltiot/bolt.py

```diff
@@ -161,11 +161,15 @@ def isAlive(self):
 
     def isOnline(self):
         """
-        Check the device status with last updated time
+        The command tells the user about the status of the device,
+        if it is Online/Offline along with time when status changed.
 
         :param None
-        :returns:  request status, device status: online/offline, time: last of last updated value
-        :example: {"success": "1", "value": "online", "time":"Sun 2018-05-06 08:14:43"}
+        :returns : {success: status of the request(true/false),
+                    value: device status (online/offline),
+                    time: the time when the device was online/offline respectively
+                          (format: day yyyy-mm-dd hh-mm-ss)}
```

Comment:
@Deepanshu625 There is a difference in the example and the docstring. It should be hh:mm:ss TZ